python/python/tests/test_async_client.py

```diff
@@ -2522,6 +2522,46 @@ async def test_zdiffstore(self, redis_client: TRedisClient):
                 await redis_client.zdiffstore("abc", ["zxy", "lkn"])
             assert "CrossSlot" in str(e)
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_zintercard(self, redis_client: TRedisClient):
+        min_version = "7.0.0"
+        if await check_if_server_version_lt(redis_client, min_version):
```

Comment:
same, I think a function for that would be good

Reply:
I'm not sure I understand, can you clarify? Are you suggesting a `skip_if_server_version_lt` function that encapsulates lines 2528 to 2530?

Reply:
I'd like to see a function which run on the very beginning and sets a global value with the version.
Furthermore, in java, C# and go clients we check binary version, not what server returns.

Reply:
@shohamazon I suggest we merge this PR for now, you can answer my questions above and then we can do it in a separate PR

Reply:
@aaron-congo yes, `skip_if_server_version_lt` sounds good and could be vey usefull